bin/sozo/src/commands/model.rs

```diff
@@ -41,6 +41,36 @@ pub enum ModelCommand {
         starknet: StarknetOptions,
     },
 
+    #[command(about = "The Dojo storage system uses the poseidon_hash function to compute \
```

Comment:
We should add a sentence to introduce the layout before diving into storage details.
"Displays the model's layout into dojo storage." and then the detailed explanation. We should also point to a documentation page (in the dojo book) to explain more in details what's happening here.